_posts/2021-06-07-How-to-edit-names-if-you-have-made-spelling-mistakes-in-the-lead-name-itself.md

```diff
@@ -0,0 +1,59 @@
+---
+date: 2021-06-07
+title: How to edit names of lead if you have made spelling mistakes in the lead name itself
+video_id: JC2yJnBXzLM
+description: Below is the steps, How to edit names of lead if you have made spelling mistakes in the lead name itself
```

Comment:
Does not seem grammatically correct. Please check with grammerly. 
Also write 2 sentences for description.